src/odemis/acq/align/test/delphi_test.py

```diff
@@ -104,8 +104,18 @@ def test_find_hole_center(self):
         data[0].shape = Y, X
 
         hole_coordinates = delphi.FindCircleCenter(data[0], 0.02032, 6, darkest=True)
-        expected_coordinates = (0.0052705, -0.0018415)  # (391.5, 257.5) px
-        numpy.testing.assert_almost_equal(hole_coordinates, expected_coordinates)
+
+        # set fixed expected coordinates for the center also set a fixed difference
+        const_diff = 6e-4
+        expected_coordinates = (0.0050, -0.0024)
+
+        # check if the outcome is between 0.0044 and 0.0056 for the first element
+        self.assertGreaterEqual(hole_coordinates[0], expected_coordinates[0] - const_diff)
+        self.assertLessEqual(hole_coordinates[0], expected_coordinates[0] + const_diff)
+
+        # check if the outcome is between -0.0030 and -0.0018 for the second element
+        self.assertGreaterEqual(hole_coordinates[1], expected_coordinates[1] - const_diff)
+        self.assertLessEqual(hole_coordinates[1], expected_coordinates[1] + const_diff)
```

Comment:
You can measure the distance between the two centers, to be a little bit more concise.
```suggestion
        # set fixed expected coordinates for the center also set a fixed difference
        max_dist = 0.5e-3 # m
        expected_coordinates = (0.0050, -0.0024)

        # check if the outcome is between 0.0044 and 0.0056 for the first element
        dist = math.hypot(hole_coordinates[0] - expected_coordinates[0],
                           hole_coordinates[1] - expected_coordinates[1])
        self.assertLessEqual(dist, max_dist)
```